### PR TITLE
Use virtuoso image directly to allow multi-arch builds.

### PR DESCRIPTION
**Pull request by brummbar**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -9,12 +9,12 @@ COPY run.sh run.sh
...
 RUN composer --no-interaction install
 
-FROM tenforce/virtuoso:1.3.1-virtuoso7.2.2
+FROM openlink/virtuoso-opensource-7:latest
 
 ENV IMPORT_DIR /tmp/import
 ENV DBA_PASSWORD dba
 
-RUN apt-get update && apt-get install -qq -y wget curl php7.0 php7.0-zip netcat
+RUN apt-get update && apt-get install -qq -y wget curl php7.4-cli php7.4-zip netcat
```
Comment (brummbar) [inline Dockerfile:17]: Investigate this in order to update to 8.1+

**Comment by brummbar** [comment]
This image doesn't let users to write into graphs. The following commands must be run:
```
GRANT SPARQL_UPDATE to "SPARQL"
DB.DBA.RDF_DEFAULT_USER_PERMS_SET ('nobody', 7);
```
[Reference 1](https://community.openlinksw.com/t/sparul-insert-access-denied-even-after-granting-update-permission/3448/6)
[Reference 2](https://github.com/openlink/virtuoso-opensource/issues/1094)

File: Dockerfile (hunk 1 above)
Comment (sergepavle) [reply Dockerfile:17]: As `openlink/virtuoso-opensource-7:latest` image is based on ubuntu 20.04 by default it shipped with php7.4. For installing 8.1+ php version we have to add PPA repository in our Dockerfile.

**Comment by sergepavle** [comment]
> This image doesn't let users to write into graphs. The following commands must be run:
> 
> ```
> GRANT SPARQL_UPDATE to "SPARQL"
> DB.DBA.RDF_DEFAULT_USER_PERMS_SET ('nobody', 7);
> ```
> 
> [Reference 1](https://community.openlinksw.com/t/sparul-insert-access-denied-even-after-granting-update-permission/3448/6) [Reference 2](https://github.com/openlink/virtuoso-opensource/issues/1094)

Sorry didn't get where we have issue with writing into graph. I didn't get how we I can reproduce issue. Our fetching and import works fine (according to builds). EWPP project with updated virtuoso seems like works good.

**Review by sergepavle — APPROVED** [review]
For now as I see it works fine.

**Comment by brummbar** [comment]
> > This image doesn't let users to write into graphs. The following commands must be run:
> > ```
> > GRANT SPARQL_UPDATE to "SPARQL"
> > DB.DBA.RDF_DEFAULT_USER_PERMS_SET ('nobody', 7);
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > [Reference 1](https://community.openlinksw.com/t/sparul-insert-access-denied-even-after-granting-update-permission/3448/6) [Reference 2](https://github.com/openlink/virtuoso-opensource/issues/1094)
> 
> Sorry didn't get where we have issue with writing into graph. I didn't get how we I can reproduce issue. Our fetching and import works fine (according to builds). EWPP project with updated virtuoso seems like works good.

Are you sure, because for me when I use `image: brummbar/triple-store-dev:latest` I got failures when running the [tests in this PR](https://github.com/openeuropa/oe_corporate_countries/pull/40).

**Comment by brummbar** [comment]
Maybe it was fixed by [looking at the changelog](https://github.com/openlink/virtuoso-opensource/blob/stable/7/NEWS.md#september-26-2023-v7211) and a fresh build.

**Review by nagyad — APPROVED** [review]

**Comment by nagyad** [comment]
Tested with EWCMS.

**Comment by brummbar** [comment]
> This image doesn't let users to write into graphs. The following commands must be run:
> 
> ```
> GRANT SPARQL_UPDATE to "SPARQL"
> DB.DBA.RDF_DEFAULT_USER_PERMS_SET ('nobody', 7);
> ```
> 
> [Reference 1](https://community.openlinksw.com/t/sparul-insert-access-denied-even-after-granting-update-permission/3448/6) [Reference 2](https://github.com/openlink/virtuoso-opensource/issues/1094)

I was correct, the current release 1.37.0 doesn't allow update queries, see #109 .

**Pull request merged**